packages/iTwinUI-react/src/core/Dialog/DialogWrapper.tsx

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { useTheme } from '../utils';
+import '@itwin/itwinui-css/css/dialog.css';
+
+export type DialogWrapperProps = React.ComponentPropsWithRef<'div'>;
+
+/**
+ * Dialog wrapper used for draggable dialogs in order to make them draggable relatively to this container.
```

Comment:
If its realtive to container, shouldnt it be limited not to go outside of the container boundaries? Otherwise, I am not sure, what it means by `relative to container`

Reply:
Also, there is this weird behavior then too:
![drag](https://user-images.githubusercontent.com/81580355/184115842-87263652-7209-49c0-8685-262c3a6d28d2.gif)

Reply:
> If its realtive to container, shouldnt it be limited not to go outside of the container boundaries? Otherwise, I am not sure, what it means by `relative to container`

I thought relative means that it can't go outside the container, e.g. can't go outside the viewer and cover header, sidenav.

Reply:
But it does go, just its not visible (why would you need that?)

Reply:
I also think it's weird. I would except the dialog's outer edges to not go outside the container.

Reply:
Dialog now can't go out of bounds, that means that bug is also gone.